app/assets/javascripts/feedback_form.js

```diff
@@ -3,6 +3,8 @@ $(document).on('turbolinks:load', function(){
   //Instantiates plugin for feedback form
 
   $("#feedback-form").feedbackForm();
+
+  console.log("instantiated feedback form");
```

Comment:
Sorry @thatbudakguy one more question. Is this leftover from development or do you want this logging to JS console in prod?

Reply:
oop, definitely left over. good catch! i found one other one too.